lib/code_coverage.rb

```diff
@@ -0,0 +1,12 @@
+require 'pathname'
+module CodeCoverage
+  # The HOOK_FILE is responsible for starting or restarting code coverage.
+  # Child worker forks inherit the code coverage environment when the server
+  # invoked the 'run_hook' so they must reset the environment for the new process.
+  HOOK_FILE = Pathname.new(__dir__).join("..", "config", "coverage_hook.rb").freeze
```

Comment:
@jamesooden note the expected file will be config/coverage_hook.rb

Reply:
@jamesooden let me know if there's a better way to word this so it's immediately obvious how this works